subsys/net/ip/tcp2_priv.h

```diff
@@ -152,7 +152,7 @@ struct tcphdr {
 	uint32_t th_ack;
```

Comment:
Could you change the commit subject to `net: tcp2: More detailed docstring`, putting the header name there is not really needed